idea_town/frontend/static-src/app/lib/page-manager.js

```diff
@@ -37,6 +41,15 @@ export default class PageManager {
       const Ctor = this.pages[page];
       const newPage = new Ctor(opts);
 
+      if (newPage.headerScroll) {
```

Comment:
I think this stuff would be better handled maybe in `base-view` using [ampersand-view's built-in subviews feature](https://github.com/ampersandjs/ampersand-view#subviews-viewsubviews).

Also, `header-view` and `scroll-header-view` look like mostly the same code. Could cut down on repetition by having the same view switch between scroll-responsive and otherwise using options in the constructor.